sonar-ws/src/main/protobuf/ws-components.proto

```diff
@@ -48,11 +48,35 @@ message ShowWsResponse {
 
 // WS api/components/suggestions
 message SuggestionsWsResponse {
-  repeated Qualifier results = 2;
-  
-  message Qualifier {
-  	optional string q = 1;
-  	repeated Component items = 4;
+  repeated Category suggestions = 2;
+  optional string warning = 3;
+  repeated Organization organizations = 4;
+  repeated Project projects = 5;
+
+  message Category {
+  	optional string category = 1;
+  	optional int64 more = 5;
```

Comment:
Done. (I thought it is not a good idea to reuse numbers - for compatibility reasons?)